recipe/meta.yaml

```diff
@@ -13,16 +13,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 200
```

Comment:
That seems incorrect to me. And if it is correct, it needs documentation.  @conda-forge/core, any thoughts?